scala/private/phases/phase_coverage.bzl

```diff
@@ -36,25 +36,28 @@ def _phase_coverage(ctx, p, srcjars):
         output_jar = ctx.actions.declare_file(
             "{}-offline.jar".format(input_jar.basename.split(".")[0]),
         )
-        in_out_pairs = [
-            (input_jar, output_jar),
+        srcs_paths = []
+        for src in ctx.files.srcs:
```

Comment:
Can we do `src_paths = [src.path for src in ctx.files.srcs]`? IMHO it's a bit cleaner and more readable to do a list comprehension than a loop with mutation.